params/config.go

```diff
@@ -109,7 +112,8 @@ type ChainConfig struct {
 	EIP155Block *big.Int `json:"eip155Block,omitempty"` // EIP155 HF block
```

Comment:
Maybe we should explain why EIP150Hash is required instead of explaining the absence of a hash field many times.